storage/mysql/admin_storage.go

```diff
@@ -98,7 +98,7 @@ func (s *mysqlAdminStorage) ReadWriteTransaction(ctx context.Context, f storage.
 }
 
 func (s *mysqlAdminStorage) CheckDatabaseAccessible(ctx context.Context) error {
-	return checkDatabaseAccessible(ctx, s.db)
+	return s.db.Ping()
```

Comment:
Probably should use PingContext(ctx) instead. Same below.

Reply:
If we're going to do this it should be PingContext()